arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -233,40 +289,32 @@
             "$ref": "#/parameters/apiVersionParameter"
           },
           {
-            "name": "resourceGroupName",
-            "description": "Name of the resource group within the Azure subscription.",
-            "in": "path",
-            "required": true,
-            "type": "string"
+            "$ref": "#/parameters/resourceGroupNameParameter"
           },
           {
-            "name": "mediaServiceName",
-            "description": "Name of the Media Service.",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "pattern": "^[a-z0-9]",
-            "minLength": 3,
-            "maxLength": 24
+            "$ref": "#/parameters/mediaResourceNameParameter"
           }
-        ]
-      },
-      "patch": {
-        "summary": "Update a Media Service.",
-        "operationId": "MediaService_Update",
+        ],
         "responses": {
           "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/MediaService"
-            }
+            "description": "Success. The Media Services account was deleted."
+          },
+          "204": {
+            "description": "No Content. The account name does not exist."
           },
           "default": {
-            "description": "Detailed error information.",
+            "description": "Delete Media Services failed.",

```

Comment:
Would this be better as "Delete Media Services failures.", or "Delete Media Services that failed", or "Delete Media Services operation failed." ?